app/models/course.rb

```
@@ -21,7 +21,7 @@ def path
   # Add a Teaching Assistant to the course
   def add_ta(user)
     if user.nil?
-      return { success: false, message: "The user with id #{user.id} does not exist" }
+      return { success: false, message: "The user with id #{user&.id} does not exist" }
```

RuboCop (1.86.1)

[convention] Redundant `return` detected.

(Style/RedundantReturn)